how-to/getting-started/how-to-buy-ether-with-usd.md

```diff
@@ -3,38 +3,50 @@ title: How to Buy Ether (ETH) with USD
 description: FAQ
 priority: 60
 date_published: '2017-06-20'
-date_modified: '2020-04-10'
+date_modified: '2020-04-21'
 ---
 
-Hello and welcome to Ethereum! Ethereum and digital currencies are pretty new and in this new financial world, things can be a little bit different.
+Hello and welcome to Ethereum! Ethereum and digital currencies are pretty new and in this new financial world, things can be a little bit different. This article gives an overview of what is ETH, where and how you can buy ETH with USD, some important info on how to store it, and how to be secure. We tried to keep it as short as possible so please take the time to read through the article. Don't hesitate to contact us directly if you have any questions or concerns, we are here to help! 
 
-First, YOU are responsible for the stuff: your investments, your choices, and most importantly, your security. There are no babysitters or undo buttons, and no bank to call up and have a transaction reversed. The power of the blockchain lies in this autonomy, but it can also be dangerous to expect all of the convenient failsafes we have  come to enjoy.
+## What IS Ether (ETH)?
 
-Second, most of the people involved in this world are pretty tech-savvy, and sometimes forget how different things are to the inexperienced. Take your time, read the instructions and warnings, ask for clarification and reach out for help if you still have questions.
+Ether, or ETH, is Ethereum's native currency. It is "digital money" that can be sent over the internet instantly and cheaply, and also be used in many Ethereum-based applications. (from [Ethereum.org](https://ethereum.org/eth/))
 
-Here are three basic security tips you need to know when buying and holding cryptocurrencies.
+## Where can you Buy ETH?
 
-* Scams, [phishing sites](https://www.google.com/safebrowsing/static/faq.html#q1), and malicious links exist. Trust messages and random links about as much as those in your spam box. Verify, verify, verify!
-* Avoid cloud storage. Always save and backup your private key in a separate location, like a piece of paper or a USB drive. 
-* Be skeptical. Does the person have an ulterior motive? Does an offer sound too good to be true?
+### MyCrypto
 
-## Still with me? Good! Let's dive deeper!
+You can go to [Buy Eth with USD via MyCrypto](https://escape.mycrypto.com/buy.html)
```

Comment:
buy.mycrypto.com has since been updated to reflect escape.mycrypto.com/buy.html, should we change this to link to buy.mycrypto.com instead?